tests/styleguide.md

```diff
@@ -0,0 +1,113 @@
+# Software for Open Networking in the Cloud - SONiC
+## Management Repo Style Guide
+
+### 1. LGTM
+PRs should not introduce any new LGTM errors, with one notable exception:
+- `no-import` errors can be ignored **if you are importing a fixture.** In this case, you should do the following:
```

Comment:
Should this read `unused-import` instead of `no-import`?

Reply:
Yep, not sure what I was looking at but you're right. Thanks!